tables/DataMan/DataManager.cc

```diff
@@ -316,7 +317,8 @@ DataManagerCtor DataManager::getCtor (const String& type)
         libname = libname.substr (0, pos);
     }
     // Try to load and initialize the dynamic library.
-    DynLib dl(libname, string("libcasa_"), "register_"+libname, False);
+    DynLib dl(libname, string("libcasa_"), STRING(CASACORE_MAJOR_VERSION),
```

Comment:
The major version should not be connected with a binary version, also bugfixes can require changing compatiblity.
If you want to couple it regardless, the next casa version would need to be 3.0.0 not 2.2.0 as there have been breaking changes merged.